subsys/bluetooth/audio/shell/bap_broadcast_assistant.c

```diff
@@ -900,6 +902,26 @@ static int cmd_bap_broadcast_assistant_add_pa_sync(const struct shell *sh,
 		}
 	}
 
+	/* use the BASE to verify the BIS indexes set by command */
+	for (size_t j = 0U; j < CONFIG_BT_BAP_BASS_MAX_SUBGROUPS; j++) {
+		if (bis_bitfield_req == 0) {
+			/* Not set the BIS indexes by command, use BASE directly */
+			break;
```

Comment:
That doesn't sound right. If the user does not want to sync to any BIS when issuing this command, we should respect that.

Reply:
in that case, subgroup_params from decoded BASE will be used directly

e.g  the command "bap_broadcast_assistant add_pa_sync true 0x0671C7"

Reply:
Yeah but if I as a user do `bap_broadcast_assistant add_pa_sync true 0x0671C7 0x00` indicating that I do not want to sync to any BIS, then that's no longer possible, right?

Reply:
0x00 is not valid index and not allowed to be used in shell command app, index should be within the range 0x01 and ox1F

Reply:
My point is that a user should be able to use this command to only request a PA sync without BIS sync (which is indicated by the `bis_sync` value of 0)